lib/viewmodels/transactions.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/services/http_service.dart';
+
+import 'package:seeds/models/models.dart';
+import 'package:seeds/viewmodels/auth.dart';
+
+class TransactionsModel extends ChangeNotifier {
+  List<Transaction> transactions;
+
+  HttpService _http;
+  AuthModel _auth;
+
+  TransactionsModel({ http, auth }) : _http = http, _auth = auth;
```

Comment:
```suggestion
  TransactionsModel({ this._http, this._auth });
```

Reply:
Well, if you change it here - you also need to change it everywhere else where class instantiated